bentoml/_internal/io_descriptors/numpy.py

```diff
@@ -22,7 +22,7 @@
 def _is_matched_shape(
     left: t.Optional[t.Tuple[int, ...]],
     right: t.Optional[t.Tuple[int, ...]],
-) -> bool:
+) -> bool:  # pragma: no cover
```

Comment:
Why no cover?

Reply:
I don't think we need to write unit test for this

Reply:
Are we even using it? What's it for?